.clang-format

```diff
@@ -0,0 +1,189 @@
+# Use clang-format 10.0 or later.
+# See https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormatStyleOptions.html
+#
+# Note: Some options are commented out as they are only available after 10.0
+#       and clang-format is not forwards-compatible (it fails on unknown keys).
+#
+---
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+#AlignConsecutiveBitFields: false
+AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: false
+AlignEscapedNewlines: DontAlign
+#AlignOperands: Align
+AlignOperands: true
+AlignTrailingComments: false
+AllowAllArgumentsOnNextLine: true
+AllowAllConstructorInitializersOnNextLine: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortBlocksOnASingleLine: Never
+AllowShortCaseLabelsOnASingleLine: false
+#AllowShortEnumsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: InlineOnly
+AllowShortIfStatementsOnASingleLine: Always
+AllowShortLambdasOnASingleLine: Inline
+AllowShortLoopsOnASingleLine: true
+AlwaysBreakAfterDefinitionReturnType: None
+AlwaysBreakAfterReturnType: None
+AlwaysBreakBeforeMultilineStrings: true
+AlwaysBreakTemplateDeclarations: Yes
+#AttributeMacros:
+#- __capability
+BasedOnStyle: Google
```

Comment:
More like Stroustrup, I believe?

Reply:
There are only a handful of builtin styles to choose from (search for BasedOnStyle on https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormatStyleOptions.html).

This setting will only take effect for options that aren't explicitly set in this config, which should be rare (I think).

Reply:
Oh, I see. Nevermind then! :)